sar_pipeline/aws/metadata/stac.py

```diff
@@ -521,10 +540,25 @@ def add_assets_from_folder(self, burst_folder: Path):
         # e.g. don't try add HH if it did not exist in original source data
         if self.product == "RTC_S1":
             pols = self.polarisations
+            calibration = self.h5.search_value(
```

Comment:
Same as above -- suggest changing this parameter to "normalization_convention" for consistency and clarity